### ISSUE TITLE
[Feature Request] Ability to change the show apps icon to custom/distro icon

### ISSUE THREAD

**Issue by n3thshan** [micheleg/dash-to-dock#2336]
### Problem
Currently the Show Applications icon is not customizable

### Proposed Solution
Add an option in the 'Appearance' or the 'Launchers' tab to allow selecting custom icons as well as some distro logos (both coloured and monochrome) already included (like in the extension 'Logo Menu') for the Show Applications button.

### Benefits and Impact
This will allow distro maintainers to give identity to their GNOME-based distro without relying on the 'Logo Menu' extension and also make the dock look nicer especially in the extended panel mode just like the Ubuntu Dock as of version 24.10.

**Comment by summoner001** [comment]
Yeah the customizable Apps icon (default grid) is a very good idea.

**Comment by D3vil0p3r** [comment]
@3v1n0 currently is not possible to change the "Show Apps" icon?
